code/game/objects/effects/effect_system/smoke.dm

```diff
@@ -257,6 +257,12 @@
 	color = "#DBCBB9"
 	smoke_traits = SMOKE_GASP|SMOKE_BLISTERING|SMOKE_OXYLOSS|SMOKE_PLASMALOSS|SMOKE_FOUL
 
+/obj/effect/particle_effect/smoke/phosphorus/mustard
+	opacity = TRUE
+	color = "#FFDB58"
```

Comment:
Why can't you have it be purple?

Reply:
Purple would overlap with tanglefoot no?